### DISCUSSION TITLE
IRAF software installation on mac os

### DISCUSSION THREAD

**Discussion by Kabirsaha001** [iraf-community/iraf#256] [Q&A]
I'm trying to install IRAF on mac m1, but after going through all the steps as suggested for mac os, I don't know whether its installed or not as its not opening. Do I need to set it up after installation? Also its saying that xgterm doesn't exit. If I need to set it up, how to do it for mac m1 ventura 13.1. Also can you suggest the steps how to access ds9 through IRAF.
Thank you

**Comment by olebole** [comment]
You also need to install the [x11iraf](https://iraf-community.github.io/x11iraf.html) package. And for ds9 access, I would recommend to read one of the tutorials. Basically, you just need to start ds9, then it can be accessed from IRAF with f.e. the **display**  or **imexamine** commands.

However, I would also like to ask you to consider the following: IRAF is really old software, nowadays lacks further development and and institutional support. It is mainly maintained for those who cannot switch yet. Unless there is a real strong requirement, it is a really bad idea to start with it for a new project. It would be much better to use modern Python projects like [Astropy](https://astropy.org/) instead whenever it is suitable. This is much more future-proof than learning an outdated, 40 year old legacy software system.
